openapi-contracts/src/main/com/sinch/sdk/domains/voice/models/dto/v1/SvamlInstructionPlayFilesDto.java

```diff
@@ -167,7 +167,7 @@ public SvamlInstructionPlayFilesDto locale(String locale) {
    * @return locale
    */
   @JsonProperty(JSON_PROPERTY_LOCALE)
-  @JsonInclude(value = JsonInclude.Include.ALWAYS)
+  @JsonInclude(value = JsonInclude.Include.USE_DEFAULTS)
```

Comment:
Waiting for the same PR on the generator repo to make sure this is not erased at the next spec update

Reply:
I'd like too: waiting for feedback from Voice team